### PR TITLE
Load curvature from file

### PR DESCRIPTION
**Pull request by finsberg**
Adding support to load curvature information from a file

### CODE REVIEW

File: smart/mesh.py [hunk 1]
```diff
@@ -177,7 +177,14 @@ def __init__(
 
         self.child_meshes = dict()
         self.parent_mesh = self
-        self.curvature = curvature
+        if isinstance(curvature, (str, Path)) and Path(curvature).is_file():
```
Comment (jorgensd) [inline smart/mesh.py:180]: Is this double clause covering the case where `Curvature=""` ?

**Review by jorgensd — APPROVED** [review]
Only minor question added.

File: smart/mesh.py (hunk 1 above)
Comment (finsberg) [reply smart/mesh.py:180]: Well, by default `curvature` is now set to `None`, which means that the double clause will cover the case when the user provides an invalid path.

**Review by emmetfrancis — APPROVED** [review]
Perfect, I was thinking of adding this in but you beat me to it!

**Pull request merged**